src/atoms/buttons/button.tsx

```diff
@@ -332,7 +348,50 @@ const ButtonIconOnly = Styled(Button)`
     };
   `}
 `;
-export const ButtonAtom:FC<ButtonProps> = ({
+const SquareButton = Styled.button<GenericStylingProps>`
+  height: 70px;
+  width: 70px;
+  border-radius: 10px;
+  border: ${({ theme }) => `2px solid ${theme.colors.primary.MAIN_BLUE}`};
+  color: ${({ theme }) => theme.textShades.SHADE_MINUS_2};
+  background: ${({ theme }) => theme.containerAndCardShades.SHADE_PLUS_2};
+  cursor: ${({ disabled }) => (disabled ? 'default' : 'pointer')}!important;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  gap: 10px;
+  font-size: 10px;
```

Comment:
i dont think font size should be restricted? i can pass a text with the proper size from the design unless there is a specific reason you have it here?